scripts/backup

```diff
@@ -1,61 +1,82 @@
 #!/bin/bash
-# Exit on command errors and treat unset variables as an error
-set -eu
 
 #=================================================
-# GENERIC STARTING
-#=================================================
-
+# GENERIC START
 #=================================================
 # IMPORT GENERIC HELPERS
 #=================================================
 
-if [ ! -e _common.sh ]; then
-    # Get the functions file if not present in the current directory
-    sudo cp ../settings/scripts/_common.sh ./_common.sh
-    sudo chmod a+rx _common.sh
-fi
-source _common.sh
+#Keep this path for calling _common.sh inside the execution's context of backup and restore scripts
+source ../settings/scripts/_common.sh
 source /usr/share/yunohost/helpers
 
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+ynh_clean_setup () {
+	ynh_clean_check_starting # Need for the param line_match of ynh_systemd_action
+}
+
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
 #=================================================
 # LOAD SETTINGS
 #=================================================
+ynh_print_info --message="Loading installation settings..."
 
 app=$YNH_APP_INSTANCE_NAME
 
-pvp=$(ynh_app_setting_get $app pvp)
-damage=$(ynh_app_setting_get $app damage)
-domain=$(ynh_app_setting_get $app domain)
-creative=$(ynh_app_setting_get $app creative)
-is_public=$(ynh_app_setting_get $app is_public)
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+domain=$(ynh_app_setting_get --app=$app --key=domain)
+
+#=================================================
+# STANDARD BACKUP STEPS
+#=================================================
+# BACKUP THE APP MAIN DIR
+#=================================================
+# STOP SYSTEMD SERVICE
+#=================================================
+ynh_systemd_action --service_name=$app --action="stop" --log_path="/var/log/$app/$app.log"
```

Comment:
https://github.com/YunoHost-Apps/minetest_ynh/blob/4648784dfcccd50aa28aaf768890770b8aca9ef2/conf/systemd.service#L10

The log path is : `--logfile /var/log/__APP__/minetest.log`

```suggestion
ynh_systemd_action --service_name=$app --action="stop" --log_path="/var/log/$app/minetest.log"
```

Reply:
line_match seems to doesn't work 
```

Warning: yunohost.hook <lambda> - [1004.1] [WARN] The service minetest didn't fully started before the timeout.
Warning: yunohost.hook <lambda> - [1004.1] [WARN] Please find here an extract of the end of the log of the service minetest:
```